milabench/datasets/fake_images.py

```diff
@@ -8,28 +8,28 @@
 
 
 def write(args):
-    image_size, count, offset, outdir = args
+    image_size, offset, count, outdir = args
     dataset = FakeData(
         size=count, image_size=image_size, num_classes=1000, random_offset=offset
```

Comment:
If we are only processing a single image, removing count from args and setting size=1 would make more sense (this might be why it was slower).

Reply:
Looking at the code for FakeData, it's not going to make any difference.